CombineUnloadAIDriver.lua

```diff
@@ -2097,6 +2097,33 @@ function CombineUnloadAIDriver:drawDebugInfo()
 
 end
 
+
+--- If the player drives the unloader, check the vicinity  for combines driven by courseplay
+--- and tell them to ignore this vehicle so their proximity sensor won't slow them down when hitting this vehicle.
+--- Note that we don't really know if the player is to unload the combine, this will disable all proximity check for
+--- player driven vehicles.
+function CombineUnloadAIDriver.disableProximitySensorForNonCourseplayDriver(unloader)
```

Comment:
`if not  g_server then return ` is missing, to make sure only the server runs this code. The client dosen't need this code as ignoreVehicleProximity should only work on the server.